test/apiv2/30-volumes.at

```diff
@@ -3,12 +3,63 @@
 # volume-related tests
 #
 
-#
-# FIXME: endpoints seem to be unimplemented, return 404
-#
-if false; then
-t GET  libpod/volumes/json            200 null
-t POST libpod/volumes/create name=foo 201
-fi
+## create volume
+volumepath=$(curl -s http://$HOST:$PORT/v1.40/libpod/info | jq -r ".store.volumePath")
```

Comment:
I'd prefer to avoid curl, or knowledge of `$HOST:$PORT`. Can you do this instead?
```diff
diff --git a/test/apiv2/30-volumes.at b/test/apiv2/30-volumes.at
index 3e7db33eb..2c38954b6 100644
--- a/test/apiv2/30-volumes.at
+++ b/test/apiv2/30-volumes.at
@@ -4,7 +4,8 @@
 #

 ## create volume
-volumepath=$(curl -s http://$HOST:$PORT/v1.40/libpod/info | jq -r ".store.volumePath")
+t GET libpod/info 200
+volumepath=$(jq -r ".store.volumePath" <<<"$output")
 t POST libpod/volumes/create name=foo1  201 \
     .Name=foo1 \
     .Driver=local \
```